tests/test_logging.py

```diff
@@ -135,14 +135,22 @@ def test_log_message_is_unchanged_if_fields_are_not_found(self):
         self.logger.info("hello {bar}")
         result = json.loads(self.buffer.getvalue())
 
-        assert result['message'] == "hello {bar}"
+        assert result['message'] == "hello {bar: missing key}"
```

Comment:
Should probably change the name of this test if we're going to do this.

Reply:
👍

Reply:
is the test name supposed to be `missing_key1` rather than `missing_key`?